WordPress/Classes/Services/BlogService.m

```diff
@@ -674,6 +679,12 @@ - (OptionsHandler)optionsHandlerWithBlogObjectID:(NSManagedObjectID *)blogObject
             }
 
             [[ContextManager sharedInstance] saveContext:self.managedObjectContext];
+            
+            if (completion) {
+                completion();
+            }
+
+            [[ContextManager sharedInstance] saveContext:self.managedObjectContext];
```

Comment:
Save Context is now being called twice (681 + 687). Not sure which one we want... probably the save call should go before completion?